src/Rings/mpoly-affine-algebras.jl

```diff
@@ -1481,7 +1481,7 @@ $A\hookrightarrow A_1\times \dots\times A_r=\overline{A}$
 of $A$ as does `normalize(A)`, but return additionally the `delta invariant` of $A$,
 that is, the dimension
 
-$\dim_K(\overline{A}/A)$.
+$\dim_K(\overline{A}/A).$
```

Comment:
Because this is actually rendered as display math (so think of this as `\[A\].` versus `\[A.\]). Hence in the current version the `.` ends up on a new line after the formula.

Reply:
Yes, but the `$` are interpreted like `\[`/`\]` in normal latex, so the dot gets printed in a newline.

Reply:
Funny, we answered in the same moment. :smile: